doc/develop/getting_started/index.rst

```diff
@@ -621,6 +621,13 @@ that are used to emulate, flash and debug Zephyr applications.
             You must rerun the setup script if you relocate the Zephyr SDK bundle directory after
             the initial setup.
 
+#. :ref:`Set these environment variables <env_vars>` as described `here
+   <toolchain_zephyr_sdk>`::
+
+   - Set :envvar:`ZEPHYR_TOOLCHAIN_VARIANT` to ``zephyr``
+   - Set :envvar:`ZEPHYR_SDK_INSTALL_DIR` to the toolchain installation
+     directory
```

Comment:
is this still a requirement with the new SDK? 

ref.
```
 65     message(STATUS "ZEPHYR_TOOLCHAIN_VARIANT not set, trying to locate Zephyr SDK")                                                                                      
```

Reply:
Hmm, good point. I was coming from the context of people switching to from GNU Arm Embedded. But still, I think it makes sense to document those or at least point users to the toolchains page? Which means documenting it there like I did.

Reply:
If everything works without these env vars, this information probably doesn't belong to the GSG, but to a section that explains more details.

Reply:
Yeah, some of us already had `ZEPHYR_TOOLCHAIN_VARIANT` set, so our old gnuarmemb install was being used, and it was unclear how to switch to the latest zephyr SDK. Possibly we didn't follow the install instructions completely enough (since we were upgrading our install, not doing a fresh install), but it would be nice to have this requirement documented somewhere for anyone who goes down the same path.

Also, apparently this was actually documented earlier and removed at some point? See https://github.com/zephyrproject-rtos/zephyr/issues/30713#issuecomment-745540137

Reply:
This is not really required anymore (the build system defaults to using the Zephyr SDK found through the CMake package).

It may be helpful to mention this in the [Zephyr SDK documentation](https://docs.zephyrproject.org/latest/develop/toolchains/zephyr_sdk.html), however.

Same goes for [GNU Arm Embeddded documentation](https://docs.zephyrproject.org/latest/develop/toolchains/gnu_arm_embedded.html).

Reply:
See the comment I've added now @stephanosio.